dev-docs/modules/audigentRtdProvider.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page_v2
+title: Audigent Segmentation RTD Module
+description: Audigent Segmentation RTD Module
+display_name: Audigent Segmentation
+module_code : audigent
+module_type : rtd
+display_name : Audigent
+enable_download : true
+sidebarType : 1
+---
+
+# Audigent Segmentation RTD Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+Audigent is a next-generation data management platform and a first-of-a-kind
+"data agency" containing some of the most exclusive content-consuming audiences
+across desktop, mobile and social platforms.
+
+This real-time data module provides quality user segmentation that can be
+attached to bid request objects destined for different SSPs in order to optimize
+targeting. Audigent maintains a large database of first-party Tradedesk Unified
+ID, Audigent Halo ID and other id provider mappings to various third-party
+segment types that are utilizable across different SSPs.  With this module,
+these segments can be retrieved and supplied to the SSP in real-time during
+the bid request cycle.
+
+### Usage
+
+Compile the audigent RTD module into your Prebid build:
+
+`gulp build --modules=userId,unifiedIdSystem,rtdModule,audigentRtdProvider,appnexusBidAdapter`
+
+
+The format of returned segments is a segment type mapping.
```

Comment:
Is knowledge of this internal detail useful to publishers? I don't think so?

I'm totally good with providing internal details in this doc, but think it should be clearly labeled:
- Focus the "Usage" section to just stuff Publishers need to do and care about
- Add a "Bidder Implementation" section if there's something bidder adapters should do.
- Add an "Implementation Details" section for internal stuff like the format of returned segments.